src/lib/utils.js

```diff
@@ -1,11 +1,36 @@
 
 import Compressor from 'compressorjs'
 import * as common from 'masq-common'
+import fetch from 'node-fetch'
```

Comment:
we are in browser's context, we can use window.fetch

Reply:
ok, done.